src/main/kotlin/com/kotlin/boardproject/domain/comment/domain/LikeComment.kt

```diff
@@ -1,5 +1,7 @@
-package com.kotlin.boardproject.model
+package com.kotlin.boardproject.domain.comment.domain
 
+import com.kotlin.boardproject.global.domain.BaseEntity
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end